docs/src/content/docs/quickstart/sql-overview.mdx

```diff
@@ -0,0 +1,581 @@
+import Txs from '@components/fiddle/txs.astro';
+import Query from '@components/fiddle/query.astro';
+import { Aside } from '@astrojs/starlight/components';
+import { Card, CardGrid } from '@astrojs/starlight/components';
+import TabbedExample from '@components/tabbed-example.astro';
+import TabCssFix from '@components/tabcssfix.astro';
+
+<TabCssFix></TabCssFix>
+
+## The Basics
+
+### Insert a row into a new table
+
+<TabbedExample>
+  <div slot="1">
+    <Aside title="'XT Play' allows you to try out XTDB live in your browser!">Feel free to edit the SQL in any of these boxes, just be aware that changes you make in one box don't carry over to the next.
+    <details style="display: contents"><summary>Details...</summary><ul>
+<li>columns are ordered alphanumerically (this restriction will be lifted [soon](https://github.com/xtdb/xt-fiddle/issues/42))</li>
+<li>you can click on the 'Open in xt-play' link beneath a given instance to see the full context and to experiment more freely</li>
+<li>all XT Play instances execute statelessly - a new ephemeral cloud database instance is summoned into being every time you press 'Run' (using AWS Lambda)</li></ul></details></Aside>
+    <Fiddle magicContext="my-context">
+      <Txs systemTime="2024-01-01"
+	   txs="INSERT INTO people (_id, name) VALUES (6, 'fred')" />
+      <Query hidden={true} q="SELECT * FROM xt.txs ORDER BY tx_time DESC LIMIT 1" />
+    </Fiddle>
+  </div>
+  <Card slot="2">
+If you don't already have XTDB running, follow the brief install steps described in [Installation via Docker](/intro/installation-via-docker).
+
+To run your first INSERT transaction, simply enter the following SQL statement into the `psql` prompt, press return, and you will see:
+
+```sql
+user=> INSERT INTO people (_id, name) VALUES (6, 'fred');
+INSERT 0 0
+```
+
+Note that XTDB doesn't currently return information about the number of rows inserted or modified by a statement when using `psql`.
+  </Card>
+</TabbedExample>
+
+Things to note:
+<ul><li>Tables in XTDB may be created dynamically during `INSERT`, where all columns and types are inferred automatically.</li><li>The only schema requirement is that every table in XTDB requires a user-provided `_id` primary key column, but all other columns are optional and dynamically typed by default. This means each row in XTDB can offer the flexibility of a document in a document database.</li><li>The `_` prefix convention (e.g. `_id`) is used for reserved columns and tables that XTDB handles automatically. For full details see [How XTDB works](/intro/data-model).</li></ul>
+
+### Query for that same row
+
+Querying this data back again is a simple matter of:
+
+<TabbedExample>
+  <div slot="1">
+    <Fiddle magicContext="my-context">
+      <Query q="SELECT * FROM people" />
+    </Fiddle>
+  </div>
+  <Card slot="2">
+```sql
+user=> SELECT * FROM people;
+ _id | name
+-----+------
+   6 | fred
+(1 row)
+```
+  </Card>
+</TabbedExample>
+
+### Evolve the table
```

Comment:
I can imagine that people will have questions about schemas here.
It would be good if we had a page with recommendations like `schema on insert` to link to here.

Reply:
Good idea - noted - will follow up on this separately